### PR TITLE
Implement Logic for Missing Glitches

### PR DESCRIPTION
**Pull request by Phantom5800**
Implementing a handful of glitches that do not currently exist in the glitch logic list.

* Volcano Sushie Glitch (Goombario + Normal Boots)
* Chapter 7 bridge hidden blocks can be hit with Super Boots frame perfectly
* Clippy in the sewers to get at the upgrade block behind metal blocks
* Dry Dry Ruins Stone Skip
* Kooper Puzzle Skip for Crystal Palace
* New Methods for Prologue Repel Gel that have been found
* Enter General Guy fight with Laki instead of Watt

### CODE REVIEW

**Review by icebound777 — COMMENTED** [review]
Note to self: Discuss with others whether Prolog Stone Block should be multiple settings instead

File: maps/graph_edges/glitched_logic/isk_ruins_stone_skip.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"""
+This file represents edges of the world graph that have to be added
+for Glitched Logic: Ruins Stone Skip.
+"""
+edges_pra_add_ruins_stone_skip= [
```
Comment (icebound777) [inline maps/graph_edges/glitched_logic/isk_ruins_stone_skip.py:5]: edges_pra should be edges_isk

File: rando_modules/modify_entrances.py [hunk 2]
```diff
@@ -869,6 +871,8 @@ def get_glitched_logic(
     # Crystal Palace
     if glitch_settings.mirror_clip:
         all_new_edges.extend(edges_pra_add_mirror_clip_laki)
+    if glitch_settings.mirror_clip:
```
Comment (icebound777) [inline rando_modules/modify_entrances.py:878]: Same settings used twice, probably meant `glitch_settings.pra_kooper_puzzle_skip`

File: rando_modules/modify_entrances.py [hunk 3]
```diff
@@ -869,6 +871,8 @@ def get_glitched_logic(
...
+        all_new_edges.extned(edges_pra_add_kooper_puzzle_skip)
```
Comment (icebound777) [inline rando_modules/modify_entrances.py:879]: Code typo, won't run

**Review by icebound777 — COMMENTED** [review]
Pointed at a few problems

**Comment by Phantom5800** [comment]
I considered adding a bunch of jumpless tricks, but may do a separate one for those in particular down the line.

**Comment by Pronyo-Chan** [comment]
lgtm

**Comment by icebound777** [comment]
Please set target branch to the dev branch instead of the main branch. Main branch is for current website state only

**Comment by Phantom5800** [comment]
Tried rebasing on dev and just ended up cherry picking commits instead because it kept pulling in bps changes from somewhere

File: maps/graph_edges/glitched_logic/omo_wattless_dark_room.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+"""
+This file represents edges of the world graph that have to be added
+for Glitched Logic: ToyBox Blue Switch Skip.
```
Comment (icebound777) [inline maps/graph_edges/glitched_logic/omo_wattless_dark_room.py:3]: referenced glitch incorrect

**Comment by icebound777** [comment]
Would you like this PR to be merged ASAP, or keep it open until we're ready for another release? In case you feel like adding more glitch settings

**Comment by Phantom5800** [comment]
This is fine to merge whenever. I wasn't planning on adding anything else.

**Comment by icebound777** [comment]
About the ruins stone skip: I seem to remember something about the padlock in that room influencing the possibility of doing this trick, or maybe making it harder if the padlock was opened already. What was up with that again?

**Comment by Phantom5800** [comment]
> About the ruins stone skip: I seem to remember something about the padlock in that room influencing the possibility of doing this trick, or maybe making it harder if the padlock was opened already. What was up with that again?

It's a lot easier to do with the lock still there because mario is able to move without laki following him as long as the exclamation point is above his head. It's possible to do in a different spot otherwise, but there isn't as good of a setup for it.

**Review by icebound777 — CHANGES_REQUESTED** [review]
Did a local checkout of the PR, found some issues

File: maps/graph_edges/glitched_logic/omo_clippy_green_station_coin_block.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+"""
+This file represents edges of the world graph that have to be added
+for Glitched Logic: Clippy Green Station Coin Block.
+"""
+edges_omo_clippy_green_station_coin_block = [
+    #* GRN Treadmills/Slot Machine -> RandomBlockItemA (MultiCoinBlock)
+    (0xA1100840, {"from": {"map": "OMO_09", "id": "ItemH"}, "to": {"map": "OMO_09", "id": "RandomBlockItemA"}, "reqs": [["Lakilester"],["Boots"]], "mapchange": False})
```
[the remark this answers is not in the document]
Comment (Phantom5800) [reply maps/graph_edges/glitched_logic/omo_clippy_green_station_coin_block.py]: this one I wasn't sure of but makes sense why boots wouldn't be required for this particular edge

**Review by icebound777 — APPROVED** [review]
`Dry Dry Ruins Stone Skip` currently does nothing in logic, due to a bug in the base graph, will fix that post merge. Otherwise lgtm

**Pull request merged**